libs/s25main/Loader.cpp

```diff
@@ -150,13 +151,19 @@ glArchivItem_Bitmap* Loader::GetNationIcon(Nation nation, BuildingType bld)
 {
     if(bld == BuildingType::Charburner)
         return LOADER.GetImageN("charburner", rttr::enum_cast(nation) * 8 + 8);
+    else if(wineaddon::isWineAddonBuildingType(bld))
+        return wineaddon::GetImage(wineaddon::GetBuildingImages(nation, bld).icon);
     else
         return convertChecked<glArchivItem_Bitmap*>(nationIcons_[nation]->get(rttr::enum_cast(bld)));
 }
 
-ITexture* Loader::GetNationTex(Nation nation, unsigned nr)
+ITexture* Loader::GetNationTex(Nation nation, BuildingType bld)
 {
-    return checkedCast<ITexture*>(GetNationImage(nation, nr));
+    if(bld == BuildingType::Charburner || wineaddon::isWineAddonBuildingType(bld))
+        return &LOADER.building_cache[nation][bld].building;
```

Comment:
Can you fix this at least for the wine-addon buildings to return the building image without the shadow.   
Or does it maybe look better with the shadow? I don't think so but we do that already for the building image in the building dialog. Maybe @Spikeone has an opinion here (whether to only use shadows for buildings on the map or also for dialogs as is now) especially compared to the original. IIRC there was even an issue long time ago, wasn't there?

Reply:
Changed to use image without shadow.